DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1925,11 +1964,19 @@ private void BroadcastGame()
             sb.Append(";");
             sb.Append(GameMode?.UntranslatedUIName ?? string.Empty);
             sb.Append(";");
-            sb.Append(tunnelHandler.CurrentTunnel.Address + ":" + tunnelHandler.CurrentTunnel.Port);
+            sb.Append(tunnelHandler?.CurrentTunnel?.Address + ":" + tunnelHandler?.CurrentTunnel?.Port);
```

Comment:
Have you test the case where `tunnelHandler?.CurrentTunnel?.Address is null`, which string will be append?

Personally I prefer if `tunnderHandler?.CurrentTunnel is null`, append `string.Empty`, otherwise append `{address}:{port}`.